src/helper.ts

```diff
@@ -84,11 +85,14 @@ export function generateConfigs(configs: ConfigsOptions = {}): string {
     }
   }
 
-  return stringify({
-    sdk_platform: 'js',
-    sdk_version: __VERSION__,
-    ...snakeCaseConfigs
-  });
+  return stringify(
+    {
+      sdk_platform: 'js',
+      sdk_version: __VERSION__,
+      ...snakeCaseConfigs
+    },
+    { indices: false, arrayFormat: 'brackets', encode: false }
```

Comment:
Will not encoding broke email and back_to url with weird chracters?

Reply:
A encoding will happen after that. Because after the config query string is done, it is stringified again with the other query params

Reply:
Will devise a test for that case